perf/replicache.ts

```diff
@@ -340,6 +330,11 @@ export function benchmarks(): Benchmark[] {
       useMemstore,
     }),
     // 128 mb is unusable
```

Comment:
whats the relevance of this comment at this point? none?

Reply:
128 is our goal so I wanted to specifically note why it's not included. I'll make that more clear.